internal/cmd/providers.go

```diff
@@ -162,6 +163,15 @@ func newProvidersRemoveCmd() *cobra.Command {
 	}
 }
 
+func GetInfraProvider() (*api.Provider, error) {
+	client, err := defaultAPIClient()
+	if err != nil {
+		return nil, err
+	}
+
+	return GetProviderByName(client, models.InternalInfraProviderName)
```

Comment:
not great that it references the models package. probably should use the string "infra"